stock-portfolio-monitor/src/client/CurrentPriceClient.ts

```diff
@@ -0,0 +1,10 @@
+import { Ticker } from "../portfolio/PortfolioState";
+
+const apiKey = process.env.REACT_APP_STOCK_API_KEY
```

Comment:
How many places do you need to change if changes come (e.g., renaming: `REACT_APP_STOCK_API_KEY` -> `STOCK_API_KEY`)?
I think, should be a single source of truth. No needs to do the same in every file. For example:
```ts
export function getApiKey(): string {
  return process.env.REACT_APP_STOCK_API_KEY;
}
```